Dockerfile

```diff
@@ -8,6 +9,8 @@ RUN apt-get update -qq && apt-get install -y \
 ADD . /app
 WORKDIR /app
 
+RUN echo ${commit_hash} > COMMIT_HASH.txt
```

Comment:
I have to wonder if there's a more standard way to embed data from the build in the image, but this is clever.

Reply:
Use of build args is pretty much standard https://docs.docker.com/compose/compose-file/#args